tests/cpgrid/inactiveCell_lgr_test.cpp

```diff
@@ -81,297 +81,305 @@ void testInactiveCellsLgrs(const std::string& deckString,
     Dune::CpGrid grid;
     grid.processEclipseFormat(&es.getInputGrid(), &es, false, false, false);
 
+    if (grid.comm().size() == 0)
```

Comment:
Is this really needed? 

As long as you do not call loadBalance on the grid, I think all the code should work in parallel, too. In this case all cells of the grid are on process 0 and for it this looks like  a serial grid. The other processes simply see an empty grid.